src/main/java/io/kontur/disasterninja/dto/eventapi/FeedEpisode.java

```diff
@@ -6,10 +6,7 @@
 import org.wololo.geojson.FeatureCollection;
 
 import java.time.OffsetDateTime;
-import java.util.HashSet;
-import java.util.Map;
-import java.util.Set;
-import java.util.UUID;
+import java.util.*;
```

Comment:
I'm investigating performance now and I recommend import concrete libs. It allows to decrease heap size